support-frontend/assets/helpers/campaigns/campaigns.ts

```diff
@@ -2,81 +2,82 @@ import type { TickerSettings } from 'components/ticker/contributionTicker';
 import type { ContributionTypes } from 'helpers/contributions';
 
 type CampaignCopy = {
-	headerCopy?: string | JSX.Element;
-	contributeCopy?: string | JSX.Element;
+  headerCopy?: string | JSX.Element;
+  contributeCopy?: string | JSX.Element;
```

Comment:
Looks like we've maybe reformatted tabs -> spaces here? The linting currently wants tabs so probably worth swapping them back